### ISSUE TITLE
PHP 7.3 gone from Ubuntu 20.04 image?

### ISSUE THREAD

**Issue by ngmy** [actions/virtual-environments#2818]
**Description**  
All of a sudden, `sudo update-alternatives --set php /usr/bin/php7.3` started to fail.

**Area for Triage**:  
PHP

**Question, Bug, or Feature?**:  
Question

**Virtual environments affected**  
- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [x] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Image version**
20210219.1

**Expected behavior**  
`sudo update-alternatives --set php /usr/bin/php7.3` succeeds.

**Actual behavior**  
`sudo update-alternatives --set php /usr/bin/php7.3` fails with exit code 2.

I ran `sudo update-alternatives --config php` for confirmation and got the following result.
```
  Selection    Path             Priority   Status
------------------------------------------------------------
* 0            /usr/bin/php8.0   80        auto mode
  1            /usr/bin/php7.4   74        manual mode
  2            /usr/bin/php8.0   80        manual mode
```

**Repro steps**  
1. Please see https://github.com/ngmy/php-enum/runs/2015665990?check_suite_focus=true

**Comment by maxim-lobanov** [comment]
Hello @ngmy , PHP 7.3 has never been installed on Ubuntu-20.04. Only on Ubuntu 16.04 and 18.04.
Your workflow has started to fail because you use `ubuntu-latest` label that is currently switching from 18.04 to 20.04
See details in https://github.com/actions/virtual-environments/issues/1816.

You have two options:
- switch back to `ubuntu-18.04`
- use `setup-php` action to install PHP on Ubuntu 20.04 in runtime: https://github.com/marketplace/actions/setup-php-action

**Comment by ngmy** [comment]
@maxim-lobanov 
I'm sorry. It was a configuration error on my part.
I fixed it to `ubuntu-18.04` and that solved it.

Thank you!
